external/cpuinfo/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+project(gflags)
+cmake_minimum_required(VERSION 3.15)
+
+include(ExternalProject)
+
+# Note that in the Bazel build the commit hash is
+# 63b254577ed77a8004a9be6ac707f3dccc4e1fd9.
+# This dependency was bumped up to 2e79955ecaec85da13ac8f1245a8b2afa10d31c2
+# in order for CMake to be able to find it.
+# Hopefully, this does not break the build.
```

Comment:
Would you mind updating the WORKSPACE to use the same commit hash of 2e79955ecaec85da13ac8f1245a8b2afa10d31c2? I think it would be better to keep the versions in lock step

```suggestion
```

Reply:
Done.